server/http.go

```diff
@@ -518,11 +518,12 @@ func handleTokenFunc(srv OIDCServer) http.HandlerFunc {
 			}
 		case oauth2.GrantTypeRefreshToken:
 			token := r.PostForm.Get("refresh_token")
+			scopes := r.PostForm.Get("scope")
```

Comment:
If the "scope" field is omitted then there's no difference from the old behavior; if they add cross-client scopes, then they'll get the cross-client behavior, provided that the refresh token was authorized for that in the first place.

We _could_ be a bit more persnickety here and require that `openid` is specified here, but I'm worried that not all clients do this, and changing behavior from previous versions. We could also check for `email` here before putting email in the claims...but we don't do that elsewhere and that would almost certainly break existing clients.